roles/opensrp/tasks/mysql.yml

```diff
@@ -9,11 +9,28 @@
     - mysql-client
     - python-mysqldb
 
+- name: Download MySQL CA certificate bundle
+  become: yes
+  become_user: root
+  get_url:
+    url: "{{ opensrp_mysql_ca_bundle_url }}"
+    dest: "/usr/local/share/ca-certificates/mysql-ca-bundle.crt"
+    force: yes
+  register: downloaded_mysql_ca_bundle
+  when: opensrp_mysql_ca_bundle_url|length > 0
+
+- name: Update MySQL CA certificates
+  become: yes
+  become_user: root
+  command: "update-ca-certificates"
```

Comment:
@ndegwamartin would this also be the same process to follow to add the CA certificates for PostgreSQL? My initial thought is yes, if the database server uses certificates that are signed by a well known CA (like RDS).

The process in https://github.com/OpenSRP/playbooks/pull/168 seems more complicated. That said, do you think it would be worthwhile to add support custom CA certificates (maybe because the CA is something we own)? What would be the best place to do that, if so?